### PR TITLE
Contribute credit

### PR DESCRIPTION
**Pull request by Harsh-Kapoorr**
I did add the "AWESOME Contributors".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,12 @@
 
 [QuickRecall](https://github.com/kaiwalyakoparkar/Data-Structures-And-Algorithm-Interview-Kit/tree/main/QuickRecall)
 
+***
+<h1 align="center">Made with ❤️ by AWESOME CONTRIBUTORS </h1>
```
Comment (kaiwalyakoparkar) [inline README.md:45]: Please convert the text into lowercase. Also, you don't need to create a new PR. Pushing changes to same branch will update the PR as well :)

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Harsh-Kapoorr) [reply README.md]: Okay will keep in mind from next time!

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

**Comment by Harsh-Kapoorr** [comment]
I'll come back again with another superpower!! :)

**Pull request merged**